src/MainEmpresaProgramadores.java

```diff
@@ -1,103 +1,135 @@
-import java.util.Scanner;
+import programacion.empresaprogramacion.Empleado;
 import programacion.empresaprogramacion.Empresa;
 import programacion.empresaprogramacion.Gerente;
 import programacion.empresaprogramacion.Programador;
+import programacion.empresaprogramacion.excepciones.ParametroInvalidoException;
+import programacion.empresaprogramacion.excepciones.RutaArchivoInvalida;
+
+import java.util.Scanner;
 
 public class MainEmpresaProgramadores {
+  public static Scanner sc = new Scanner(System.in);
 
   public static void main(String[] args) {
 
-    Empresa empresa = new Empresa("Empresa del Juan de la Cierva");
-    int opcion = 0;
-
-    Scanner scanner = new Scanner(System.in);
-
-    referencia = new Programador(parametros);
-    empresa.addEmpleado(referencia);
-
-    referencia = new Programador(parametros);
-    empresa.addEmpleado(referencia);
-
-    referencia = new Gerente(parametros);
-    empresa.addEmpleado(referencia);
-
-    referencia = new Gerente(parametros);
-    empresa.addEmpleado(referencia);
-
-    referencia = new Programador(parametros);
-    empresa.addEmpleado(referencia);
-
-    referencia = new Programador(parametros);
-    empresa.addEmpleado(referencia);
-
-    referencia = new Programador(parametros);
-    empresa.addEmpleado(referencia);
-
-    referencia = new Programador(parametros);
-    empresa.addEmpleado(referencia);
+    Empresa empresa = new Empresa("Tecnologías Gyllenhaal");
+    int opcion;
+
+    Empleado referencia;
+
+    try {
+      referencia = new Programador("50549668S", "Daniel", 1302.93, "Java", true);
+      empresa.addEmpleado(referencia);
+    } catch (ParametroInvalidoException e) {
+      System.out.println(e.getMessage());
```

Comment:
Si sólo imprimes el mensaje de la excepción, podría ser poca información. Se me ocurre algo así:
```suggestion
      System.out.println("Excepción por ParametroInvalido creando Programador: " + e.getMessage());
```